modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -10,21 +10,87 @@ image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
-When you deploy an application to CloudHub, you have access to the validated OS updates, security patches, and bug fixes that MuleSoft provides regularly.
 
-You can keep your CloudHub applications up to date by manually updating them at your convenience.
-If you don't update apps manually, MuleSoft applies the updates automatically.
+To ensure the security and stability of CloudHub, MuleSoft releases security updates
+and monthly patches and applies them automatically on a schedule.
 
-When a new runtime version is available for your CloudHub application, the *Applications* page displays a banner and the *Update Available* tab lists the apps that require updates:
+Automatic updates use blue-green deployment, which ensures zero downtime.
```

Comment:
@hannanelson Can we change this to something like below, (i am not sure about the tense and refer to the customer as you):
CloudHub is an integration platform as a service (iPaaS), which means that the patch and upgrades of the OS and the mule software are managed for you by the platform. This means you can focus on developing and updating the application functionality.

CloudHub applies security patching to ensure your application is secure. CloudHub applies monthly mule updates to ensure you application continues to be stable. All patching and updates use blue-green deployment, which ensures zero downtime.